docs/source/development/traffic_portal.rst

```diff
@@ -45,5 +45,11 @@ Installing The Traffic Portal Developer Environment
 #. Run ``npm install`` to install application dependencies into ``traffic_portal/node_modules``. Only needs to be done the first time unless ``traffic_portal/package.json`` changes.
 #. Run ``bower install`` to install client-side dependencies into ``traffic_portal/app/bower_components``. Only needs to be done the first time unless ``traffic_portal/bower.json`` changes.
 #. Run ``grunt`` to package the application into ``traffic_portal/app/dist``, start a local HTTPS server (Express), and start a file watcher.
-#. Valid SSL certificates and keys are needed for Traffic Portal to run. Generate these (e.g. using `this SuperUser answer <https://superuser.com/questions/226192/avoid-password-prompt-for-keys-and-prompts-for-dn-information#answer-226229>`_) and edit the ``traffic_portal/conf/config.js`` file to reflect their names and locations
+#. Modify ``traffic_portal/conf/config.js``:
+
+  #. Valid SSL certificates and keys are needed for Traffic Portal to run. Generate these (e.g. using `this SuperUser answer <https://superuser.com/questions/226192/avoid-password-prompt-for-keys-and-prompts-for-dn-information#answer-226229>`_) and update ``ssl``.
+  #. Modify ``api.base_url`` to point to your Traffic Ops API endpoint.
+  #. Modify ``files.static`` to be ``./app/dist/public``.
+  #. Modify ``log.stream`` to be ``./server/log/access.log``.
```

Comment:
You're using spaces for indentation instead of tabs. Not only is this the greatest sin anyone can commit, but it changes the semantic meaning of your indentation - which is part of the reason tabs are mandated.

What you have here is a list, followed by a blockquote that contains a second list, followed by a third list. This renders (in HTML output) like:
```html
<ol>
	<li>Some upper step</li>
	<li>Modify <code>traffic_portal/conf/config.js</code></li>
</ol>
<blockquote>
	<div>
		<ol>
			<li>Modify this</li>
			<li>Modify that</li>
		</ol>
	</div>
</blockquote>
<ol>
	<li>Final step</li>
</ol>
```
If you use tabs instead you get proper nested lists like:
```html
<ol>
	<li>Some upper step</li>
	<li>Modify <code>traffic_portal/conf/config.js</code>
		<ol>
			<li>Modify this</li>
			<li>Modify that</li>
		</ol></li>
	<li>Final step</li>
</li>
```
(Actually there's still a `div` and a `blockquote` in there, but the nested structure will be correct)

Reply:
Because of this you've got something that goes:
1. step
2. step
3. step
4. step
5. step
6. step

<blockquote><ol><li>substep</li><li>substep</li><li>substep</li></ol></blockquote>

1. step